internal/provisioner/plain/main.go

```diff
@@ -165,15 +167,31 @@ func main() {
 		os.Exit(1)
 	}
 
+	const plainBundleProvisionerName = "plain"
```

Comment:
does it make sense to move constant in the same package that contains the plain provisioner's unique ID constant variable?